src/components/Dropdown/DropdownMenuButton/DropdownMenuButton.stories.tsx

```diff
@@ -27,12 +27,12 @@ const Template: React.FC<Omit<ComponentProps<typeof DropdownMenuButton>, 'childr
 
 export const Default: Story = () => (
   <Cluster align="center">
-    <Template />
-    <Template disabled />
-    <Template onlyIconTrigger />
+    <Template label="デフォルト" />
+    <Template disabled label="無効化" />
+    <Template onlyIconTrigger label="アイコンのみ" />
```

Comment:
props の説明をするならラベル以外でしたい気持ち。

```suggestion
    <Template label="その他の操作" />
    <Template disabled label="その他の操作" />
    <Template onlyIconTrigger label="その他の操作" />
```